app/src/main/java/com/github/godspeed010/weblib/fragments/WebViewFragment.kt

```diff
@@ -201,10 +201,18 @@ class WebViewFragment : Fragment() {
     }
 
     private fun scrollWebView(progression: Float, wv: WebView?) {
+        // Fragment has been closed, no need to scroll web view anymore.
+        if(view == null)
+            return;
```

Comment:
```suggestion
        if (view == null) return
```
Semicolons aren't needed in Kotlin. I also just prefer to leave the code on the same line as if statement if they're both really short.